.ci/generate-releases.sh

```diff
@@ -0,0 +1,214 @@
+#!/usr/bin/env bash
```

Comment:
This implementation uses the existing logic defined in https://github.com/elastic/apm-pipeline-library/blob/main/updatecli/updatecli.d/bump-aliases.yml

But creating different files that will be uploaded with a GitHub action to the given google bucket